src/javascripts/menu items/forms/formModal.js

```diff
@@ -0,0 +1,20 @@
+
+const formModal = (modalTitle) => {
```

Comment:
nit: If you have a components folder, any function that just returns HTML should live within that folder. You can separate your components out by feature there.